### PR TITLE
[2019-11-16] New Characters/Weapons 

### PR DESCRIPTION
**Pull request by OrkunKocyigit**
**Scope of Change:**

This PR aims to add following weapons/characters.
https://gbf.wiki/Jeanne_d%27Arc_(Grand)
https://gbf.wiki/Shitori
https://gbf.wiki/Krugne_(SR)
https://gbf.wiki/Sacred_Standard
https://gbf.wiki/Iridescent_Bow
https://gbf.wiki/Cube_Staff

**Changes included in this PR**

- [ ] Jeanne d'Arc has been added to charData.
- [ ] Jeanne d'Arc support skill "%10 Bonus Dmg on TA for Light Allies" has been implemented.
- [ ] Jeanne d'Arc support skill "%30 Elemental Boost on Element Buff" has been implemented.
- [ ] Shitori has been added to charData.
- [ ] Krugne has been added to charData.
- [ ] Sacred Standard has been added to armData.
- [ ] Iridescent Bow has been added to armData.
- [ ] Cube Staff has been added to armData.

**Changes NOT included in this PR**
- Shitori support skill that boosts bow characters.
- Dark-Opus skill refactor for same reason as #375

### CODE REVIEW

**Comment by Groovman** [comment]
Need to do some reordering of weapons and characters then I merge it, changes were good.

File: armData.json [hunk 1]
```diff
@@ -19254,6 +19254,86 @@
...
+        "type": "bow",
+        "skill1": "normalKatsumiM",
+        "skill2": "non",
+        "element2": "light",
+        "skill3": "non",
+        "element3": "light",
+        "minhp": 38,
+        "minattack": 417,
+        "hp": 269,
+        "attack": 3142,
```
Comment (kagekiyo7) [inline armData.json]: >         "hp": 269,
>         "attack": 3142,

Is this status true? I could not find reliable information.
([4★] Iridescent Bow)

File: armData.json [hunk 2]
```diff
@@ -25531,5 +25611,24 @@
         "maxlv": 75,
         "imageURL": "./imgs/1039900000.png",
         "en": "Fury Stone"
+    },
+    "キューブ・スタッフ": {
+        "ja": "キューブ・スタッフ",
+        "element": "wind",
+        "type": "wand",
+        "skill1": "non",
+        "skill2": "normalM",
```
Comment (kagekiyo7) [inline armData.json]: `"skill1": "normalM",` is true.
(no need to fix)

File: src/global_logic.js [hunk 3]
```diff
@@ -2860,8 +2860,12 @@ module.exports.treatSupportAbility = function (totals, chara, buff) {
                     totals[key]["ougiDamageLimitBuff"] += support.value;
                     continue;
                 case "additionalDamageXA":
-                    // currently, range: own only, and no chances to stack.
-                    totals[key]["additionalDamageXA"] = support.value;
+                    // Implemented Stacking. Can someone verify?
```
Comment (kagekiyo7) [inline src/global_logic.js]: Stacking:
Support ability of Levin Sisters and Jeanne d'Arc (Grand) does not coexist. (Levin Sisters is adding 15% when TA.)
https://gbf.wiki/Levin_Sisters

 info:
https://xn--bck3aza1a2if6kra4ee0hf.gamewith.jp/article/show/176457

Comment (Groovman) [reply src/global_logic.js]: reverted till verified

**Pull request merged**